libs/state/src/checkout/order/order-state.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { StoreModule } from '@ngrx/store';
+
+import { reducers } from './reducers/index';
+import { EffectsModule } from '@ngrx/effects';
+import { OrderEffects } from './effects/order.effects';
+import { CartEffects } from '../../cart/effects/cart.effects';
+
+@NgModule({
+  imports: [
+    StoreModule.forFeature('order', reducers),
+    EffectsModule.forFeature([
+      OrderEffects,
+      CartEffects
```

Comment:
Why does this need to define `CartEffects`?

Reply:
I had it there, because the effect to handle clearCart was in CartEffects. I moved it to OrderEffects though, so this was removed